extra/artifacts/_common.json

```
@@ -46,6 +46,11 @@
       "packager": "arduino",
       "name": "remoteocd",
       "version": "0.0.4-rc.4"
+    },
+    {
+      "packager": "arduino",
```

I see that tool is published under your own packager, so it should be:

```suggestion
      "packager": "SiliconLabs",
```